app/assets/stylesheets/spotlight/_pages.scss

```diff
@@ -102,6 +102,9 @@
 }
 
 .item-grid-block {
+  margin-left: -15px;
+  margin-right: -15px;
```

Comment:
Not sure if there is a variable that would be better to use here.  The other thing that might be a possibility would be to keep the `row` class but also add an explicit `d-block`, but not sure if that's any more straight forward.

Reply:
Maybe `-($grid-gutter-width / 2)`?